boards/nxp/frdm_mcxa156/doc/index.rst

```diff
@@ -175,6 +175,12 @@ should see the following message in the terminal:
    *** Booting Zephyr OS build v3.6.0-4478-ge6c3a42f5f52 ***
    Hello World! frdm_mcxa156/mcxa156
 
+Troubleshooting
+===============
+
+.. include:: ../../common/segger-ecc-systemview.rst
```

Comment:
is there any way to do this other than a relative path, I dont think there is any guarantee this folder structure will always be the same, if we have all these peppered relative paths in things it will be really hard if there is some restructuring later

Reply:
From the [sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#id4):

> The file name is usually relative to the current file’s path. However, if it is absolute (starting with /), it is relative to the top source directory.

However every example I see in tree uses a relative path:
```
 grep -r "\.\. include:" --include="*.rst"
boards/up-bridge-the-gap/up_squared_pro_7000/doc/up_squared_pro_7000.rst:.. include:: ../../../intel/common/efi_boot.rst
boards/up-bridge-the-gap/up_squared_pro_7000/doc/up_squared_pro_7000.rst:.. include:: ../../../intel/common/net_boot.rst
boards/up-bridge-the-gap/up_squared/doc/index.rst:.. include:: ../../../../soc/intel/apollo_lake/doc/supported_features.txt
boards/up-bridge-the-gap/up_squared/doc/index.rst:.. include:: ../../../intel/common/efi_boot.rst
boards/up-bridge-the-gap/up_squared/doc/index.rst:.. include:: ../../../intel/common/net_boot.rst
boards/nxp/mimxrt1060_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/nxp/mimxrt1015_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/nxp/mimxrt1020_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/nxp/mimxrt1010_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/nxp/mimxrt1024_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/nxp/mimxrt1064_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/nxp/mimxrt1050_evk/doc/index.rst:.. include:: ../../common/opensda-debug.rst
boards/nxp/mimxrt1050_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/nxp/mimxrt1040_evk/doc/index.rst:.. include:: ../../common/rt1xxx-lpclink2-debug.rst
boards/intel/rpl/doc/index.rst:.. include:: ../../../../soc/intel/raptor_lake/doc/supported_features.txt
boards/intel/rpl/doc/index.rst:.. include:: ../../../intel/common/efi_boot.rst
boards/intel/adl/doc/index.rst:.. include:: ../../../intel/common/efi_boot.rst
boards/intel/ehl/doc/index.rst:.. include:: ../../../../soc/intel/elkhart_lake/doc/supported_features.txt
boards/intel/ehl/doc/index.rst:.. include:: ../../../intel/common/efi_boot.rst
boards/intel/ehl/doc/index.rst:.. include:: ../../../intel/common/net_boot.rst
boards/intel/ish/doc/index.rst:.. include:: ../../../../soc/intel/intel_ish/doc/supported_features.txt
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/gen_syscalls.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/gen_device_deps.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/gen_kobject_list.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/gen_offset_header.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/parse_syscalls.py
doc/build/cmake/index.rst:.. include:: ../../../arch/x86/gen_idt.py
doc/build/cmake/index.rst:.. include:: ../../../arch/x86/gen_gdt.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/gen_relocate_app.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/process_gperf.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/gen_app_partitions.py
doc/build/cmake/index.rst:.. include:: ../../../scripts/build/check_init_priorities.py
doc/introduction/index.rst:.. include:: ../../README.rst
```

I'm not sure why this folder structure would change? If it did, the doc build would fail and we'd fix up the paths. We also store this file within the `boards/nxp` folder anyways, so I think we generally control the structure/naming of these folders

Reply:
the folder structure changed earlier this year, IDK

Reply:
True- I don't expect we'll have a rework as large as hardware model V2 again any time soon though